src/parser.cpp

```diff
@@ -636,9 +643,16 @@ void Parser::ParserImpl::loadUnit(const UnitsPtr &units, const XmlNodePtr &node)
             }
         } else if (attribute->isType("multiplier")) {
             if (isCellMLReal(attribute->value())) {
-                multiplier = convertToDouble(attribute->value());
+                if (!convertToDouble(attribute->value(), multiplier)) {
+                    // TODO This value won't be saved for validation later, so it does need to be reported now
```

Comment:
Fullstop here too.

Reply:
(and below)

Reply:
Yes, my bad for just copying/pasting things... :) Done in commit 39420217.